package.json

```diff
@@ -49,7 +50,9 @@
     "ember-notify": "~5.2.1",
     "ember-resolver": "^4.0.0",
     "ember-source": "~2.18.0",
-    "loader.js": "^4.2.3"
+    "jsdom": "^11.6.2",
+    "loader.js": "^4.2.3",
+    "prember": "0.3.0-alpha.3"
```

Comment:
This is an alpha because it depends on unreleased PRs in ember-cli-fastboot and broccoli-asset-rev. I won't be mad if you don't want to merge until these are real releases.